### PR TITLE
Vendorize administration.

### PR DESCRIPTION
**Pull request by chadwcarlson**
## What's changed

Vendorize administration section, including changes to manage users and pricing.

- [x] CLI, including installation instructions (https://github.com/platformsh/platformsh-docs/issues/3473)
- [x] Console
- [x] Organizations
- [x] Teams - not available at MVP launch, hidden from sidebar using temp `api:curl` commands
- [x] Users
- [x] Pricing
- [x] Server upgrades

Closes https://github.com/platformsh/platformsh-docs/issues/3473 with https://github.com/platformsh/platformsh-docs/pull/3469/commits/1680dee2eafd3280c45953e38000bd758981ac0d

## Changes for Upsun external review

- [Administration](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration.html)
   - [CLI](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/cli.html)
      - [API Tokens](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/cli/api-tokens.html) 
   - [Console](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/web.html)
      - [Configure a project](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/web/configure-project.html)
      - [Configure environments](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/web/configure-environment.html)
      - [MFA ](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/web/mfa.html)
   - [Organizations](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/organizations.html)
   - [Users](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/users.html)
   - [Teams (hidden)](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/teams.html)
   - [Pricing](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/pricing.html)
   - [Server upgrades](https://docs.upsun.com.context-admin-twdxtoi-652soceglkw4u.eu-3.platformsh.site/administration/servers.html)

### CODE REVIEW

File: sites/platform/src/administration/web/_index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: |
 
 {{% description %}}
 
-![Overview of projects in the Console](/images/management-console/all-projects.png "0.5")
+![Overview of projects in the Console](/images/management-console/all-projects-revised.png "0.5")
```
Comment (AnouckColson) [inline sites/platform/src/administration/web/_index.md:11]: l.17: link to the console needs to be vendorized

File: sites/platform/src/administration/users.md [hunk 2]
```diff
@@ -239,10 +239,13 @@ you can invite other users to your organization and grant them the following per
 - **List projects** (`projects:list`):
   See all projects in an organization, even those the user can't access.
 
-{{< note theme="warning" >}}
+{{< note theme="info" >}}
 
 Users with the **Manage users** (`members`) permission can add, edit, or remove _any_ user's permissions except their own.
 
+Users with the **Manage billing** (`billing`) permission automatically are granted **List projects** (`projects:list`) permissions.
```
Comment (AnouckColson) [inline sites/platform/src/administration/users.md:246]:
```suggestion
Users with the **Manage billing** (`billing`) permission automatically are granted **List projects** (`projects:list`) permission. 
```

File: sites/friday/src/administration/pricing.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+The [Pricing page](https://upsun.com/pricing/) should always be considered the primary source of pricing details.
+
+During the {{% vendor/name %}} Beta phase, certain features are reduced in cost or even free, that will change after the Beta has finished. 
+Features may be included during the Beta phase that will become paid features outside of Beta.
+
+Make sure to consult the [Pricing page](https://upsun.com/pricing/) for exact details related to the Beta feature availability and pricing.
+
+{{< /note >}}
+
+In general, there are four main dimensions that determine the cost of work on {{% vendor/name %}}
```
Comment (AnouckColson) [inline sites/friday/src/administration/pricing.md:23]:
```suggestion
In general, there are four main dimensions that determine the cost of work on {{% vendor/name %}}.
```

File: sites/friday/src/administration/pricing.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+|  Pricing dimension  |  Details  |
+|---|---|
+|  Project fees |  An individual project on {{% vendor/name %}} comes with a consistent monthly cost. The cost includes all of the orchestration and provisioning characteristics that defines the {{% vendor/name %}} PaaS.<br/><br/>It comes with some baseline features and feature limits, such as infrastructure metrics and a certain number of build minutes. Individual preview environments do not come with their own associated creation cost - you are charged only for the resources those preview environments use during their lifespan (see next row).  |
```
Comment (AnouckColson) [inline sites/friday/src/administration/pricing.md:27]:
```suggestion
|  Project fees |  An individual project on {{% vendor/name %}} comes with a consistent monthly cost. The cost includes all of the orchestration and provisioning characteristics that define the {{% vendor/name %}} PaaS.<br/><br/>It comes with some baseline features and feature limits, such as infrastructure metrics and a certain number of build minutes. Individual preview environments do not come with their own associated creation cost - you are charged only for the resources those preview environments use during their lifespan (see next row).  |
```

**Review by AnouckColson — DISMISSED** [review]

File: sites/platform/src/administration/web/_index.md (hunk 1 above)
Comment (chadwcarlson) [reply sites/platform/src/administration/web/_index.md:11]: Fixed!

**Review by AnouckColson — APPROVED** [review]

**Pull request merged**